_posts/2023-11-02-release.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Release 2023.44
+author: Priya
+date: 2023-11-02
+category: Jekyll
+layout: post
+---
+### Enhancements:
+
+#### Date Difference Calculation - Now with System Time and Specific Date Options:
+Calculate date differences more effectively with additional choices. You can choose between system time, specific dates, and even set calculations to the current date-time in UTC. The date difference component is also conveniently displayed in pipelines (Years, Months, Weeks, etc).
+
+#### Insert Current Date and Time - Now with UTC Option:
```

Comment:
### Label and Insert rule now allows inserting current time in date column
The label and insert rule has now been updated to allow for inserting the current time besides any static time that one could previously enter through the calendar widget. The current time is re-evaulated every time the specific label insert task run.